### PR TITLE
Project Completion

### PR DESCRIPTION
**Pull request by dodoburner**
My partners @Alfred-KInyua, @NickEmma, and I @dodoburner have finished the catalog of my things console app with the requirements specified at https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md. The video presentation is in the README description.

### CODE REVIEW

File: item.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :genre, :author, :label
```
Comment (prolajumokeoni) [inline item.rb:4]: - [ ] Based on the requirements, kindly make Genre, Author, and Label, in your item class to be read-only 👍🏿 

```
attr_reader :genre
attr_reader :author
attr_reader :label
```

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Alfred-KInyua,  @dodoburner and @NickEmma, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [x] Kindly record another video using zoom or another platform as some requirements for the [project documentation](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#project-documentation) is not yet met because the audio for the other two team members is off, please after recording another video following all the requirements, kindly make sure that you watch to check that there is clear audio for all team members👍🏿

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: item.rb (hunk 1 above)
Comment (nickemma) [reply item.rb]: @prolajumokeoni 
Thank you for your review. noted i will do that.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi Team,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by nickemma** [comment]
Thank you for reviewing and approving our code.

**Pull request merged**